pkg/validations/upgradevalidations/preflightvalidations.go

```diff
@@ -22,7 +22,7 @@ func (u *UpgradeValidations) PreflightValidations(ctx context.Context) (err erro
 		upgradeValidations,
 		validations.ValidationResult{
 			Name:        "validate taints support",
-			Remediation: "ensure TAINTS_SUPPORT env variable is set",
+			Remediation: "ensure TAINTS_SUPPORT env variable is set to true",
```

Comment:
This is not something we want to mention right? If the env var is not set, it should just throw the error as if there is no support for taints at all. But let me know if there is another example you got this from.

Reply:
That is how the messages for the other checks constructed. You can find the other messages in the same file.